### PR TITLE
Feature/redshift

### PR DESCRIPTION
**Pull request by EugenRakhimov**
source
This pull request adds ability to connect to Redshift simultaneously with main database. Basically, it's pretty much the same code, so I may refactor if I got a point into the right direction.

### CODE REVIEW

File: db/redshift.go [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+	_ "github.com/lib/pq"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/cenkalti/backoff"
+)
+
+var (
+
+	// connRedhshift is the current redshift connection
+	connRedhshift *sql.DB
```
Comment (dylannz) [inline db/redshift.go]: `connRedshift`

File: db/redshift.go [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+	// onceRedshift prevents InitConnection from being called more than once in Conn
+	onceRedshift = sync.Once{}
+)
+
+// InitConnectionRedhshift creates a new new connection to the database and verifies that it succeeds.
+func InitConnectionRedhshift() {
```
Comment (dylannz) [inline db/redshift.go]: `InitConnectionRedshift`

File: db/redshift.go [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+	db := RS{}
+	db.Open()
+	connRedhshift = db.Conn
+
+}
+
+// SetConnection manually sets the connection.
+func SetConnectionRedhsift(db *sql.DB) {
```
Comment (dylannz) [inline db/redshift.go]: `SetConnectionRedshift`

File: db/redshift.go [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+		password,
+		env.GetString("REDSHIFT_HOST", "localhost"),
+		env.GetString("REDSHIFT_PORT", "5439"),
+		env.MustGetString("REDSHIFT_NAME"),
+		env.GetString("REDSHIFT_SSL_MODE", "disable"),
+	)
+
+	searchPath := env.GetString("REDSHIFT_SEARCH_PATH", "")
+	if len(searchPath) > 0 {
+		connString = fmt.Sprintf("%s&search_path=%s", connString, searchPath)
```
Comment (dylannz) [inline db/redshift.go]: Does redshift support search_path parameter?

File: db/redshift.go [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+	// connRedshift is the current redshift connection
+	connRedshift *sql.DB
+
+	// onceRedshift prevents InitConnection from being called more than once in Conn
+	onceRedshift = sync.Once{}
+)
+
+// InitConnectionRedshift creates a new new connection to the database and verifies that it succeeds.
+func InitConnectionRedshift() {
+	db := RS{}
```
Comment (dylanmacdonald) [inline db/redshift.go]: Should be something like 'rs', looks like you're using the package name as a variable

**Pull request merged**